src/main/webapp/app/entities/submitted-answer/submitted-answer-dialog.html

```diff
@@ -15,7 +15,13 @@ <h4 class="modal-title" id="mySubmittedAnswerLabel" data-translate="artemisApp.s
         </div>
 
         <div class="form-group">
-            <label data-translate="artemisApp.submittedAnswer.submission" for="field_submission">Submission</label>
+            <label data-translate="arTeMiSApp.submittedAnswer.question" for="field_question">Question</label>
```

Comment:
I think it always has to be "artemisApp" instead of "arTeMiSApp". Please search and replace all occurrences!